code/datums/jobs.dm

```diff
@@ -2189,6 +2189,7 @@ ABSTRACT_TYPE(/datum/job/special/halloween/critter)
 		..()
 		if (!M)
 			return
+		M.bioHolder.RemoveEffect("buzz")
```

Comment:
this really looks like a bandaid and not the actual fix, just make random genes not apply to ghost respawn shit like this

Reply:
Makes sense, could a solution to this be to do M.bioHolder.RemoveAllPoolEffects()? Looking at how traits work it seems as though they're applied before getting to the job setup code.

Reply:
I think I have the fix now, I'm deactivating anything that was activated from effectsPool prior to spawn, and I extended the fix to the rest of the halloween critters too.